src/react-components/help-dialog.js

```diff
@@ -12,23 +12,14 @@ export default class HelpDialog extends Component {
             Use your controller&apos;s action button to teleport from place to place. If it has a trigger, use it to
             pick up objects.
           </p>
-          <p style={{ textAlign: "center" }}>
-            In VR, <b>look up</b> to find your menu:
-            <img
-              className="info-dialog__help__hud"
-              src="../assets/images/help-hud.png"
-              srcSet="../assets/images/help-hud@2x.png 2x"
-            />
-          </p>
           <p>
-            The <b>Mic Toggle</b> mutes your mic.
+            In VR, <b>look up</b> to find your menu.
           </p>
           <p>
-            The <b>Pause/Resume Toggle</b> pauses all other avatars. You can then block them from having further
-            interactions with you.
+            The <b>Mic Toggle</b> mutes your mic.
           </p>
           <p>
-            The <b>Bubble Toggle</b> hides avatars that enter your personal space.
+            The <b>Pause/Resume Toggle</b> pauses all other avatars and lets you block others or remove objects.
```

Comment:
Should we change the media "Delete" buttons to read "Remove" instead? There are three of them in the HTML: https://github.com/mozilla/hubs/blob/feature/new-coat-of-paint/src/hub.html#L171

Reply:
ah yes, i meant to do this! thanks